include/morphio/properties.h

```diff
@@ -146,6 +146,7 @@ struct Annotation {
 struct Marker {
     PointLevel _pointLevel;
     std::string _label;
+    int32_t _parentId;  // id of parent section
```

Comment:
so, classic question: is this 0 the soma, or not?  We need to be careful about this due to the soma not being considered a section...

Reply:
Please elaborate here. `_parentId` can be -1 for cases when a marker belongs to the soma or marker belongs to another marker. See https://github.com/BlueBrain/MorphIO/blob/master/src/readers/morphologyASC.cpp#L146 and https://github.com/BlueBrain/MorphIO/blob/master/src/readers/morphologyASC.cpp#L141

Reply:
I think that's offset from what is returned from `morphology::section`; section_id == 0 is a neurite, not the soma.

Reply:
> _parentId can be -1 for cases when a marker belongs to the soma

Ah, ok, pardon me, that seems right.